packages/block-editor/src/components/block-settings-menu/block-edit-visually-button.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+import { rawHandler, getBlockContent } from '@wordpress/blocks';
+import { ToolbarButton } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { store as blockEditorStore } from '../../store';
+
+export default function BlockEditVisuallyButton( { clientIds, ...props } ) {
+	const { block, shouldRender } = useSelect(
+		( select ) => {
+			const firstBlockClientId = clientIds[ 0 ];
+			const { isBlockMultiSelected, getBlockMode, getBlock } = select(
+				blockEditorStore
+			);
+			const isSingleSelected = ! isBlockMultiSelected(
+				firstBlockClientId
+			);
+			const isHtmlMode = getBlockMode( firstBlockClientId ) === 'html';
+
+			return {
+				block: getBlock( firstBlockClientId ),
+				shouldRender: isSingleSelected && isHtmlMode,
+			};
+		},
+		[ clientIds[ 0 ] ]
+	);
+
+	const { replaceBlocks } = useDispatch( blockEditorStore );
+	const onClick = useCallback( () => {
+		replaceBlocks(
+			block.clientId,
+			rawHandler( { HTML: getBlockContent( block ) } )
+		);
+	}, [ block ] );
+
+	if ( ! shouldRender ) {
+		return null;
+	}
+
+	return (
+		<ToolbarButton
+			title={ __( 'Edit visually' ) }
```

Comment:
I don't think there needs to be a `title` prop, since the button already contains a visible label.
```suggestion
```

Reply:
@ZebulanStanphill if we remove that line as you suggested, `aria-label="Edit visually"` will not be added to the button. Not sure do we want to do that

Reply:
There's no need for an [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) on a button that already has a visible label. The whole point of `aria-label`s is to provide labels visible to screen readers for buttons that didn't actually have a readable label (usually because the button uses icons rather than text). In this case, the button is already text, so any additional label attribute is entirely redundant.

Reply:
@ZebulanStanphill  I see. OK, thanks for the explanation. I've just removed that line.